### DISCUSSION TITLE
Option to separate followed Hashtags and followed Accounts in the Home feed.

### DISCUSSION THREAD

**Discussion by mudd258** [mastodon/mastodon#26357] [General]
There should be a way to separate the accounts and hashtags I am following because the accounts I am following usually gets buried within posts that come from tags on the Home feed.

**Comment by kwfer23** [comment]
With the new release, you can now create a list where the feed is excluded from the home feed. If I could just add a hashtag to a list, this would solve the problem. The hashtag posts would go into the list feed. I could also group multiple, related hashtags this way! And, the followed hashtags wouldn't be in such an obscure place in the web interface.

So, feature request to allow followed hashtags in lists?

**Reply by mudd258** [reply]
Sounds like a viable solution. It seems like there is already an issue filed for this [here](https://github.com/mastodon/mastodon/issues/7692).
